product-service-ts/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "lib": ["es2017"],
```

Comment:
What do you think, do you need tests in deployed Lambda code?

Why do you need to build tests and deploy to AWS? Maybe this is not needed and even bad practice?

Please split folders to src/ (only code base) and test/ (only tests)

Reply:
I don't think that I need tests to be deployed. Thank you.

I have split code into 2 folders in such a way: 
![image](https://user-images.githubusercontent.com/8244819/97924428-4ff20f00-1d68-11eb-81c9-db41b7fbb22d.png)

Is it ok?